install_gae.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex  # Exit on error; debugging enabled.
```

Comment:
It could be a bad idea to put this into your environment.  Since you are sourcing this file, it will affect everything from here on.

You could do things in a single chained command instead:
```
mkdir \
&& curl \
&& unzip \
&& export
```